dash/backend/src/modules/user/enum/Authority.ts

```diff
@@ -4,6 +4,7 @@ export enum Authority {
     READ_ONLY = 'READ_ONLY',
     KUBEHUNTER = 'KUBEHUNTER',
     KUBEBENCH = 'KUBEBENCH',
-    TRAWLER ='TRAWLER',
-    FALCO = 'FALCO'
+    TRAWLER = 'TRAWLER',
+    FALCO = 'FALCO',
+    API_KEY = 'API_KEY',
```

Comment:
This authority level should probably be called 'Metrics' or something similar.

It is very strange to have most API Keys not have the API Key level, and the Authority Level API KEY granting you access to the metrics endpoint is not intuitive.

Reply:
We don't really have anywhere else we use api keys. In my mind, this was a way to start granting them access to things.

The UI attaches API Keys to specific user profiles. This approach is more of a service-based API Key: giving other programs access to endpoints.

Reply:
I can change it so it's more specific. I just wanted to start making a path forward for programmatic access to endpoints.

Reply:
Instead of having to make an endless number of Authorities based on specific criteria ;)